src/items_types.h

```diff
@@ -22,7 +22,7 @@ typedef enum
     UNKNOWN = 0,
     KILOGRAMS,
     LITERS,
-    UNITS
+    UNITS,
```

Comment:
For future reference, all enums should probably have some sort of prefix, since their name is global. Otherwise you might run into a situation where a common name like `UNKNOWN` can mean several different thing